### PR TITLE
Remove Uncyclopedia specifc checks

### PR DESCRIPTION
**Pull request by macbre**
Thanks to [PLATFORM-1055](https://wikia-inc.atlassian.net/browse/PLATFORM-1055) Uncyclopedia uses shared users database. We can remove these checks / comments from our code.

@wladekb / @michalroszka

### CODE REVIEW

File: extensions/wikia/UserRenameTool/RenameUserHelper.class.php [hunk 1]
```diff
@@ -10,9 +10,7 @@ class RenameUserHelper {
 
 	const CLUSTER_DEFAULT = '';
 
-	public static $excludedWikis = array(
-		425, /* uncyclopedia */
-	);
+	public static $excludedWikis = [];
```
Comment (hakubo) [inline extensions/wikia/UserRenameTool/RenameUserHelper.class.php]: is it edited dynamically?
maybe we could remove the var all together?

Comment (macbre) [inline extensions/wikia/UserRenameTool/RenameUserHelper.class.php]: Good point, let's leave the code a bit better than it was :)

71b709c

**Comment by ghost** [comment]
:+1:

File: extensions/wikia/UserRenameTool/RenameUserHelper.class.php (hunk 1 above)
Comment (hakubo) [inline extensions/wikia/UserRenameTool/RenameUserHelper.class.php]: :heart:

**Pull request merged**